src/main/app/src/app/formulieren/taken/model/aanvullende-informatie.ts

```diff
@@ -88,21 +92,9 @@ export class AanvullendeInformatie extends AbstractTaakFormulier {
       this.informatieObjectenService.listEnkelvoudigInformatieobjecten(
         zoekparameters,
       );
-    const morgen = new Date();
-    morgen.setDate(morgen.getDate() + 1);
-    const fataledatumZaak =
-      this.zaak.uiterlijkeEinddatumAfdoening &&
-      new Date(this.zaak.uiterlijkeEinddatumAfdoening);
-    let fataleDatumBuilder = new DateFormFieldBuilder(
-      this.humanTaskData.fataledatum,
-    )
-      .id(AbstractTaakFormulier.TAAK_FATALEDATUM)
-      .minDate(morgen)
-      .label("fataledatum")
-      .showDays();
-    if (fataledatumZaak) {
-      fataleDatumBuilder = fataleDatumBuilder.maxDate(fataledatumZaak);
-    }
+
+    const tomorrow = new Date(new Date().setDate(new Date().getDate() + 1));
```

Comment:
We're using moment already. Wouldn't this be more readable using moment?

Reply:
The expected type is Date

```  minDate(date: Date) {
    this.formField.minDate = date;
    return this;
  }```